.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "files.associations": {
+        "hmac_sha256.h": "c",
+        "system_error": "c",
+        "xlocmon": "c",
+        "xtr1common": "c"
+    }
+}
```

Comment:
I believe this file should not exist here, or the folder it is kept in

Reply:
Agreed. I'll delete this